content/graphing/infrastructure/hostmap.md

```diff
@@ -12,114 +12,118 @@ further_reading:
+## Overview
 
-Below is a subset of Datadog's infrastructure. As you can see, **c3.2xlarge** instances are pretty heavily loaded.
+Host Maps visualize hosts together on one screen, with metrics made comprehensible via color and shape.    
 
-{{< img src="graphing/infrastructure/hostmap/hostmappart1image2.png" alt="host map part 1" responsive="true" style="width:80%;">}}
+## Usage
 
-As seen below, by clicking on the c3.2xlarge group and then sub-grouping by role, we found that only some of the roles are loaded, while others are nearly idling. If we downgraded those 7 green nodes to a c3.xlarge, we would save almost $13K per year. That's worth investigating! ( $0.21 saved per hour per host x 24 hr/day * 365 days/year * 7 hosts = $12,877.20 / year )
+### Filter by
 
-{{< img src="graphing/infrastructure/hostmap/hostmappart1image3.png" alt="Datadog Host Maps Instance-Role Groups" responsive="true" style="width:80%;">}}
+`Filter by` limits the Host Map to a specific subset of an infrastructure. The filter input bar in the top left enables filtering of the Host Map by tags as well as Datadog-provided attributes.
 
-### Availability Zone Placement
+If the filter input bar is empty, the map displays all hosts that are reporting metrics to Datadog.
 
-Host maps make it easy to see distributions of machines in each of your availability zones (AZ). Filter for the hosts you are interested in, group by AZ, and you can immediately see whether resources need rebalancing. As seen below, there is an uneven distribution of hosts with `role:daniels` across availability zones. (Daniels is the name of an internal application.)
+Example: if you tag your hosts by the environment they are in, you can filter by 'production' to remove hosts in your staging and other environments from the map. If you want to eliminate all but one host role in production, then add that role to the filter, too—the filters are `AND`ed together.
 
-{{< img src="graphing/infrastructure/hostmap/hostmappart1image4.png" alt="Datadog Host Maps AZ Balance" responsive="true" style="width:80%;" >}}
+Filterable host attributes (automatically provided):
 
-### Problem Investigation
+* `up`: the host is reporting a heartbeat
+* `down`: the host is not reporting a heartbeat
+* `muted`: Datadog alerts are muted for this host
+* `agent`: the host is running the Datadog Agent
+* `agent_issue`: often indicates an integration problem, such failed access to a resource
+* `upgrade_required`: the Datadog Agent requires an upgrade
```

Comment:
Checked with DataViz, this list is not available in the host map. Lines 30-37 should be deleted.